drivers/gs.go

```diff
@@ -156,6 +156,20 @@ func (os *gsSession) createClient() error {
 	return nil
 }
 
+func (os *gsSession) DeleteFile(ctx context.Context, name string) error {
+	if !os.useFullAPI {
+		return errors.New("delete not supported for non full api")
+	}
+	if os.client == nil {
+		if err := os.createClient(); err != nil {
+			return err
+		}
+	}
+	return os.client.Bucket(os.bucket).
+		Object(name).
```

Comment:
We should probably join the session key with the filename provided here as well for consistency with other methods

Reply:
Done (I think). Let me know if that's correct